docs/plans/2026-05-10-ontology-2-5-0-polyglot-ingestion.md

```
@@ -0,0 +1,575 @@
+
+**Dependencies:** Task 13, Task 20
+**Parallelizable:** Yes (independent of Task 21)
+**testingStrategy:** integration
+
+---
+
+## PR-A acceptance gate
+
+Run before opening PR-A:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)